dmutils/forms.py

```diff
@@ -39,7 +39,7 @@ class EmailValidator(Regexp):
 
     def __init__(self, **kwargs):
         kwargs.setdefault("message", "Please enter a valid email address.")
-        super(EmailValidator, self).__init__(self._email_re, **kwargs)
+        return super(EmailValidator, self).__init__(self._email_re, **kwargs)
```

Comment:
Why do we need this return?

Reply:
We don't, but all I've done here is exactly revert the previous commits, taking us back to exactly where we were before. This should make the changes less contentious, but... /shrug